glue/viewers/image/state.py

```diff
@@ -567,9 +569,11 @@ def _update_syncing(self, *args):
         if self.global_sync:
             self._sync_color.enable_syncing()
             self._sync_alpha.enable_syncing()
+            # self._sync_preferred_cmap.enable_syncing()
```

Comment:
Can you remove lines like this that are commented out?

Reply:
Yup, done.